docs/source/revision_history.rst

```diff
@@ -4,6 +4,19 @@ Revision history
 List of major changes (the `Mypy Blog <http://mypy-lang.blogspot.com/>`_ contains more
 detailed release notes):
 
+- September 2018
+    * Publish ``mypy`` version 0.630 on PyPI.
+
+      * Recategorize and document ``--warn-incomplete-stub`` (:ref:`docs <warn-incomplete-stub>`).
```

Comment:
Style nit: The term "recategorize" is unclear. Maybe just leave it out and rewrite as "Document ...".